data/map systems.txt

```diff
@@ -7583,7 +7595,9 @@ system Caeculus
 	trade Medical 508
 	trade Metal 272
 	trade Plastic 289
-	fleet "Korath Ember Waste Raid" 20000
+	fleet "Kor Alaji Raid" 40000
+	fleet "Kor Isaha Raid" 50000
+	fleet "Korath Ember Waste Raid" 60000
```

Comment:
Is there a reason why raid chances are being increased here?

Reply:
I did some recalculation based on non-keystone JD distance from Korath space.  Don't remember exactly which systems changed but this is likely that.